### PR TITLE
add workflow to deploy docs to GitHub pages

### LINKED ISSUE #81: Publish docs to webpages
Create actions to publish documentation pages, before the repository is made publicly available:

- [x] MkDocs 
- [x] Doxygen

Problem: As far as I know, GitHub Pages only supports one page.

### PR DESCRIPTION
**Pull request by jarkenau**
- add theme for `doxygen`
- add workflow which builds and publishes `mkdocs` and `doxygen` to the `gh-pages` branch, so that it can be used by GitHub pages

closes #81

### CODE REVIEW

**Comment by mhoellmann** [comment]
I changed the visibility to public and pointed the github pages to the gh-pages branch. 

https://agri-gaia.github.io/seerep/

File: docs/gh-pages/index.html
Comment (mhoellmann) [inline docs/gh-pages/index.html:11]: If I'm not mistaken you can remove the "index.html" at the end

**Comment by jarkenau** [comment]
@mhoellmann is there anything I should add to the Doxygen main page? It's kinda short, but I can't think of anything else.

**Review by mhoellmann — APPROVED** [review]

**Comment by jarkenau** [comment]
I had to merge it again, because the merge button was sill not available for me, after you merged it. :neutral_face:

**Pull request merged**
